src/components/details/_details.scss

```diff
@@ -34,18 +35,13 @@
     text-decoration: underline;
   }
 
-  .govuk-details__summary:hover {
-    color: $govuk-link-hover-colour;
+  // Remove the underline when focussed to avoid duplicate borders
+  .govuk-details__summary:focus .govuk-details__summary-text {
+    text-decoration: none;
   }
 
-  .govuk-details__summary:focus {
-    // -1px offset fixes gap between background and outline in Firefox
-    outline: ($govuk-focus-width + 1px) solid $govuk-focus-colour;
-    outline-offset: -1px;
-    // When focussed, the text colour needs to be darker to ensure that colour
-    // contrast is still acceptable
-    color: $govuk-focus-text-colour;
-    background: $govuk-focus-colour;
+  .govuk-details__summary:hover {
+    color: $govuk-link-hover-colour;
```

Comment:
This is affecting focus styles too, which I don't believe it should be.

![Screenshot 2019-05-03 at 09 35 16BST](https://user-images.githubusercontent.com/121939/57126985-d9490f80-6d86-11e9-9279-19579ec3e62c.png)